### DISCUSSION TITLE
poorly done documentation

### DISCUSSION THREAD

**Discussion by jose1937JS** [wix/react-native-ui-lib#1808] [Ideas]
It is not for nothing bad, currently I had to use its library because it was already integrated into a project and the truth is that it is difficult to get the project out, I try to read how it works in its documentation, the components it has and its properties and I feel that it **A lot of information is needed, it really is incomplete.**

**Comment by ethanshar** [comment]
Hi @jose1937JS 
Thank you for the feedback. 

It would really help if you can point specific issues with our docs.
What are you missing? what do you wish to be different?

**Comment by manikanta500** [comment]
Yes very poor documentation. Very disappointed. 
Points that i dint like 
- 1. whever i click to check the component, i cant preview immediately, it takes time to load the preview of the component. sometimes some coponents are not even showing up in the docs of that component. only the props and other details are visible. If u see other library docs , each component is easily visualisable and no such loading time.
- 3. The usage example is so bad just a line of code. When i copy paste the same code in my system, its not showing up, or looks different that what i see in the preview. and if i try to follow that example u give at the top, it redirects to the github page which is not so nice(usually the libraries ive seen have their example in the same page along with the code and preview in 2 tabs. its not good to always get redirected for everything. Bad user experience.
- 4. Again even those example dont have some description of what is the example doing , even copy paste this example doesnt give expected result. The example should have a preview so that we can understand whats going on.( u can preview in github)
- 5. again each component should have some good enough description. with each props having good description, with examples if possible.
- 6. overall there should be so much previews to visuallize things ,this helps understanding quick, not spending so much time reading everything, trying out and understanding. So much time waste. 
- 7. you examples and preview should be such that we can copy paste as it is and be able to see the same result. But ur examples are incomplete. There is no import statements, no export default function etc.. no return.. no props ,.. no data.. just the component code is not enough. A complete code that can be copy pastable is perfect. You should refer other existing standard libraries and make a good docs only then it will be user friendly.
